apps/docs/app/features/code-block/CodeBlock.tsx

```diff
@@ -26,7 +33,7 @@ export const CodeBlock = ({
         {...defaultProps}
         theme={theme}
         code={code}
-        language={language}
+        language={language as any}
```

Comment:
`Highlight`-komponenten til `prism-react-renderer` støtter ikke arbitrære strenger, så må omgå denne for å ikke få en typefeil for å sende inn `elm`.